pkg/cri/server/container_create.go

```diff
@@ -332,6 +333,11 @@ func (c *criService) volumeMounts(containerRootDir string, criMounts []*runtime.
 		}
 		volumeID := util.GenerateID()
 		src := filepath.Join(containerRootDir, "volumes", volumeID)
+		if !filepath.IsAbs(dst) && goruntime.GOOS != "windows" {
+			oldDst := dst
+			dst = filepath.Join("/", dst)
+			log.L.Infof("Volume destination %q is not absolute, converted to %q", oldDst, dst)
```

Comment:
debug pls

Reply:
I kept log.L, as we don't have a context here, but changed it to Debugf